modules/nf-core/genomad/endtoend/meta.yml

```diff
@@ -98,3 +95,4 @@ authors:
   - "@CarsonJM"
 maintainers:
   - "@CarsonJM"
+  - "@gallvp"
```

Comment:
Do you want to be a maintainer on these modules (several others too)?

Reply:
Good point. This is from an automated script which adds it if the `yml` is modified. In this context, it is too ambitious. I'll comment it out. Thank you!